manager/assets/modext/widgets/resource/modx.panel.resource.js

```diff
@@ -53,21 +72,13 @@ Ext.extend(MODx.panel.Resource,MODx.FormPanel,{
             } else if (pcmb) {
                 pcmb.setValue(this.config.record.parent_pagetitle+' ('+this.config.record.parent+')');
             }
-            if (!Ext.isEmpty(this.config.record.pagetitle)) {
-                var title = Ext.util.Format.stripTags(this.config.record.pagetitle);
-                title = Ext.util.Format.htmlEncode(title);
-                if (MODx.perm.tree_show_resource_ids) {
-                    title = title + (this.config.record.id ? ' <small>('+this.config.record.id+')</small>' : '');
-                }
-                Ext.getCmp('modx-header-breadcrumbs').updateHeader(title);
-            }
+
+            this.formatMainPanelTitle('resource', this.config.record);
+
             // initial check to enable realtime alias
-            if (Ext.isEmpty(this.config.record.alias)) {
-                this.config.aliaswasempty = true;
-            } else {
-                this.config.aliaswasempty = false;
-            }
-            this.config.translitloading = false; // the initial value for the realtime-alias throttling
+            this.config.aliaswasempty = Ext.isEmpty(this.config.record.alias) ? true : false ;
```

Comment:
You seem to like ternary conditions a lot.
```suggestion
            this.config.aliaswasempty = Ext.isEmpty(this.config.record.alias);
```